### PR TITLE
 Use UniValue.pushKV rather than push_back(Pair())

### PR DESCRIPTION
**Pull request by sickpig**
This is basically a scripted diff:

```sh
$> grep -Fr "push_back(Pair" src/ | awk -F ':' '{print $1}' | sort | uniq | xargs -I {} sed --in-place -e 's/push_back(Pair/pushKV/' {}
$> grep -IFr pushKV src/ | awk -F ':' '{print $1}' | sort | uniq | xargs -I {} sed --in-place -e '/pushKV/s/))/)/' {}
```

This include also a port of  bitcoin/bitcoin#14164 which removes the deprecated std::pair wrappers from univalue.

### CODE REVIEW

**Comment by sickpig** [comment]
`rpc_rawsign` is failing, looking at it now.

File: src/univalue/test/object.cpp [hunk 1]
```diff
@@ -239,33 +239,33 @@ BOOST_AUTO_TEST_CASE(univalue_object)
 
     strKey = "age";
     v.setInt(100);
-    BOOST_CHECK(obj.pushKV(strKey, v));
+    BOOST_CHECK(obj.pushKV(strKey, v);
```
Comment (ptschip) [inline src/univalue/test/object.cpp]: looks to me the right parenthesis has gone missing in all of these tests?

Comment (sickpig) [reply src/univalue/test/object.cpp]: you are right, the script I used got to aggressive :P

Comment (sickpig) [reply src/univalue/test/object.cpp]: fixed

**Review by ptschip — APPROVED** [review]

**Pull request merged**